R/methods-element.R

```diff
@@ -1,6 +1,7 @@
 #' @title Add Data Collection Method Element 
 #' @description Adds the data collection method information of a dataset based off of EML standards. 
-#' @param method_file Please provide a word document with proper formatting. The 
+#' @param parent_element Please provide a parent element to append methods to. 
+#' @param methods_file Please provide a word document with proper formatting. The 
 #' function will then append it properly. An example word document can be found in 
 #' "~/EDIutils/inst/extdata/methods-template.docx". This file can be opened
```

Comment:
do we want to be instructing people to modify data within the package like this? my bias is no

Reply:
I agree. I will remove sentence that describes changing the word doc and look at the other functions to remove instructions that ask people to modify data within a package.